rest/blip_sync_messages.go

```diff
@@ -17,41 +18,75 @@ type SequenceIDParser func(since string) (db.SequenceID, error)
 
 // Helper for handling BLIP subChanges requests.  Supports Stringer() interface to log aspects of the request.
 type subChangesParams struct {
-	rq               *blip.Message    // The underlying BLIP message
-	logger           base.SGLogger    // A logger object which might encompass more state (eg, blipContext id)
-	zeroSeq          db.SequenceID    // A zero sequence ID with correct subtype (int sequence / channel clock)
-	sequenceIDParser SequenceIDParser // Function which can convert a sequence id string to a db.SequenceID
+	rq      *blip.Message // The underlying BLIP message
+	_since  db.SequenceID // Since value on the incoming request
```

Comment:
I can't remember -- what do the underscores mean?

Reply:
Internal only - in this case they are needed to differentiate between the struct property and the pre-existing accessors (since(), docIDs()).  The accessors aren't really doing any work at this point, but I left them in for consistency with the rest of the changes message parameters.